EIPS/eip-2386.md

```diff
@@ -0,0 +1,208 @@
+---
+eip: 2386
+title: Walletstore
```

Comment:
Please use a title which includes Eth2.0 or BLS12-381 in it, otherwise it is easy to mistake it with Eth1.

Reply:
Updated; thanks.